backend/src/apps/api/v1/organizations.py

```diff
@@ -1,15 +1,30 @@
 from django.urls import path
-from rest_framework import generics
+
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import generics, status
 from rest_framework.permissions import IsAuthenticated
 
+from apps.core.views.base import BaseOrganizationModelView, BaseOrganizationDetailView
 from apps.api.serializers.organizations import OrganizationSerializer
 from apps.organizations.models import Organization, Role, Membership
 from apps.organizations.permissions import HasRole
 
 
-class OrganizationsAPI(generics.ListCreateAPIView):
+@extend_schema(tags=["Организации"])
+@extend_schema_view(
+    post=extend_schema(
+        summary="Создание организации",
+        request=OrganizationSerializer,
+        responses={status.HTTP_201_CREATED: OrganizationSerializer},
+    ),
+    get=extend_schema(
+        summary="Получение списка всех организаций пользователя",
+        responses={status.HTTP_200_OK: OrganizationSerializer},
+    ),
+)
+class OrganizationsAPI(BaseOrganizationModelView):
```

Comment:
Зачем? `BaseOrganizationModelView` - отвечает за правильную выборку объектов в рамках одной организации. Организация не может принадлежать организации.

Нужно вернуть как было

```suggestion
class OrganizationsAPI(generics.ListCreateAPIView):
```